### PR TITLE
Change YUM package manager for DNF

### PR DESCRIPTION
**Pull request by aranajhonny**
Update linux.md

### CODE REVIEW

**Comment by thedaniel** [comment]
Could you comment with more context for this change?

**Comment by mnquintana** [comment]
@thedaniel AFAIK `yum` has been deprecated in favor of `dnf` as the de facto, default package manager for RPM-based Linux distros. 

However, I'm only familiar with this change for Fedora - was `yum` replaced with `dnf` in CentOS and RHEL too @aranajhonny?

File: docs/build-instructions/linux.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Ubuntu LTS 12.04 64-bit is the recommended platform.
 
 ### Fedora / CentOS / RHEL
 
-* `sudo yum --assumeyes install make gcc gcc-c++ glibc-devel git-core libgnome-keyring-devel rpmdevtools`
+* `sudo dnf --assumeyes install make gcc gcc-c++ glibc-devel git-core libgnome-keyring-devel rpmdevtools`
```
Comment (kevinsawicki) [inline docs/build-instructions/linux.md]: This is qualified in the main readme as a fedora 21 vs. fedora 22+

https://github.com/atom/atom#red-hat-linux-fedora-21-and-under-centos-red-hat

Maybe we should do the same here?

**Comment by aranajhonny** [comment]
DNF offers some backwards compatibility with Yum. you’ll be reminded that yum is deprecated and that you should use the dnf command instead.

**Comment by kevinsawicki** [comment]
@aranajhonny is `dnf` available in Fedora 21?

**Comment by aranajhonny** [comment]
@kevinsawicki yes if from fedora 18.I before using fedora 22 not use much time in fedora 21
captura de pantalla 2015-08-18 a la s 16 49 07

**Comment by kevinsawicki** [comment]
> @kevinsawicki yes if from fedora 18.I before using fedora 22 not use much time in fedora 21

Great, thanks for the details.

**Comment by aranajhonny** [comment]
@kevinsawicki thanks, Have an Octotastic day!

**Comment by kevinsawicki** [comment]
>  thanks, Have an Octotastic day!

I will, you do the same :octocat: :sunglasses:

**Pull request merged**